app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @recipe.name %></h1>
+<p>Complexity: <%= @recipe.complexity %></p>
+<p>Genre: <%= @recipe.genre %></p>
+
+<% @recipe_ingredients.each do |ingredient| %>
+<p>Ingredient name: <%= ingredient.name %></p>
+<p>Total Cost: <%=  %></p>
```

Comment:
I think this is also close, but note that total cost isn't something we would call on a `recipe_ingredient` or even an `ingredient` for that matter, so we could call it outside this `.each` block.